### PR TITLE
Highlight dont put

### PR DESCRIPTION
**Pull request by anonoz**
This PR updates ruby and gem versions for vuln fix reasons, and add a panel around dont put.

I didn't complete setting up my local development environment, @dannyongtey I need your help to help me test out if my simple changes work in practice.

### CODE REVIEW

**Comment by dannyongtey** [comment]
PR is tested locally and working fine. Going to merge.

**Comment by anonoz** [comment]
email no markdown

**Comment by dannyongtey** [comment]
Markdown added.

**Comment by dannyongtey** [comment]
Just noticed, the pie chart statistic doesn't work

**Comment by anonoz** [comment]
Make an issue, I think my mass `bundle update` accidentally bumped major version of some gems/libraries and caused regression.

File: Gemfile.lock [hunk 1]
```diff
@@ -1,271 +1,280 @@
...
-    byebug (9.0.6)
-    callsite (0.0.11)
-    chartkick (1.4.2)
-    climate_control (0.0.3)
-      activesupport (>= 3.0)
+    builder (3.2.4)
+    byebug (11.0.1)
+    case_transform (0.2)
+      activesupport
+    chartkick (3.3.1)
```
Comment (anonoz) [inline Gemfile.lock:75]: @dannyongtey Try put `~> 1.0` for chartkick in gemfile? Bumped major by 2 sure break API.

**Review by anonoz — COMMENTED** [review]
K lah I suspect this, that's why almost all gems in Gemfile should have pessimistic versioning `~>`

**Comment by anonoz** [comment]
So lets say if ppl really chiong in and sign in, maybe need to put a bare minimum "Don't Put Photo and Address" somewhere/everywhere.

File: Gemfile.lock (hunk 1 above)
Comment (dannyongtey) [reply Gemfile.lock:75]: okay, lemme try.

**Pull request merged**